model_drafts/tiles/edge.js

```diff
@@ -0,0 +1,29 @@
+var Sequelize = require('sequelize');
+
+function Edge(db){
+  var Edge = db.define('Edge', {
+    type: Sequelize.STRING,
```

Comment:
If we're using degrees (0,90,180,270), should we change this to integer? Or do we want to still refer to edges as NSEW and refer to direction as degrees?